yellowbrick/contrib/missing/base.py

```diff
@@ -60,8 +60,8 @@ def fit(self, X, y=None, **kwargs):
 
         self.y = y
 
-        super(MissingDataVisualizer, self).fit(X, y, **kwargs)
```

Comment:
We probably do still want the call to `super` right? To ensure that the target type is identified and that the class and feature names are collected?

Reply:
Yup! I think that dealing with this in #923 will make more sense.